ytmusicapi/parsers/browsing.py

```diff
@@ -54,13 +54,23 @@ def parse_content_list(results, parse_func, key=MTRIR):
 def parse_album(result):
     return {
         "title": nav(result, TITLE_TEXT),
-        "year": nav(result, SUBTITLE2, True),
+        "type": nav(result, SUBTITLE0),
+        "artists": [parse_id_name(x) for x in nav(result, ["subtitle", "runs"]) if "navigationEndpoint" in x],
         "browseId": nav(result, TITLE + NAVIGATION_BROWSE_ID),
+        # making previously unavailable keys snakecase
+        "audioPlaylistId": nav(result, THUMBNAIL_OVERLAY, True),
         "thumbnails": nav(result, THUMBNAIL_RENDERER),
         "isExplicit": nav(result, SUBTITLE_BADGE_LABEL, True) is not None,
     }
 
 
+def parse_id_name(sub_run):
```

Comment:
I intentionally left it generic as the same pattern appears outside of artists. Didn't commit any related changes in this PR, but just about anything with a title and browse id could be pulled with it. That being said, its more than likely a weird duplicate of some other function I didn't see.

Reply:
There's not yet another function like it, so that is fine. Functions with similar parsing: `parse_song_flat`, `parse_song_artist_runs`, `parse_song_runs`

`parse_song_runs` and `parse_song_artist_runs` are places where you could use this function as well. I suggest refactoring these functions to use it

`parse_song_flat` is an example where you'd parse an `album` name and id - which works differently. Therefore I'd at least include the word `artist` in the function name: `parse_artist_id_name` - as artist is the only context we know where it works